### DISCUSSION TITLE
Is it possible to install fastAPI on Raspberry Pi without docker?

### DISCUSSION THREAD

**Discussion by moritzsalla** [fastapi/fastapi#7154] [Questions] [answered]
Installing the fastapi package with pip will throw this error:

```
pi@raspberrypi:~/Desktop/ $ pip install fastapi

DEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.
Defaulting to user installation because normal site-packages is not writeable
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
ERROR: Could not find a version that satisfies the requirement fastapi (from versions: none)
ERROR: No matching distribution found for fastapi
```

**Comment by luebke-dev** [comment]
Install a newer python version

https://installvirtual.com/how-to-install-python-3-8-on-raspberry-pi-raspbian/

the most recent version is 3.9. just replace the 3.8 with 3.9 and you have the most recent version running.

**Comment by moritzsalla** [comment]
I'm on python 3.8 and pip 20.2.3.

**Comment by Mause** [comment]
Make sure you're using the right pip on your system - try `pip3.8`

**Comment by ycd** [comment]
@moritzsalla, please run these and post the outputs here.

```python
python -V
```

```python
pip -V
```

```python
python3 -V
```

```python
pip3 -V
```

**Comment by moritzsalla** [comment, marked as answer]
I had an SSL certificate issue with my python install. Thanks for the help!

**Comment by tiangolo** [comment]
Thanks for the help everyone! 🍰 

And thanks for coming back to close the issue @moritzsalla ☕ 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
